.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/tsickle
+    docker:
+      - image: alexeagle/ngcontainer
+    steps:
+      - checkout
+      - restore_cache:
+          key: tsickle-{{ .Branch }}-{{ checksum "yarn.lock" }}
```

Comment:
It's a SHA256: https://circleci.com/docs/2.0/configuration-reference/#save_cache
(it's not an arbitrary unix command, it's part of their config DSL)